### DISCUSSION TITLE
Is there an API for managing Config Revision?

### DISCUSSION THREAD

**Discussion by ihumster** [netbox-community/netbox#17561] [Q&A]
Perhaps I did not carefully look at the documentation and research the API, but I do not see any namespace for managing Config Revision of Netbox.

My interest is in developing CUSTOM VALIDATORS and automatically delivering them to Netbox. I would like to manage configuration updates in a single CI/CD process.

**Comment by ihumster** [comment]
@jeremystretch May I draw your attention to this thread?

**Reply by jeremystretch** [reply]
No.
